intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -39,6 +39,7 @@ We support the following models:
 ### 1. Build LLM Runtime
 Linux
 ```shell
+git submodule update --init --recursive
```

Comment:
I think there is no submodule under the graph.

Reply:
If we don't “python setup.py install” for  ITREX , we will be missing pybind11, which will result in an error during  only compile graph.

Reply:
Split pybind compile and cpp-only compile in the future